packages/block-editor/src/components/url-popover/link-editor.js

```diff
@@ -20,6 +20,8 @@ export default function LinkEditor( {
 	autocompleteRef,
 	className,
 	onChangeInputValue,
+	isFullWidth,
+	hasBorder,
```

Comment:
`<LinkEditor isFullWidth hasBorder />` would make me think the `LinkEditor` gets a border and becomes full width, but really it's the `URLInput` that gets this. I think the `LinkEditor` props could be named something like `hasInputBorder`, `isFullWidthInput`.